faker/providers/company/it_IT/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .. import Provider as CompanyProvider
+from faker.utils.checksums import calculate_luhn
```

Comment:
Could you move this import to the top of the file, and put a newline between absolute and relative imports: https://travis-ci.org/github/joke2k/faker/jobs/667152889#L234-L241